### PR TITLE
Update project configuration files

### PR DESCRIPTION
**Pull request by nisarg73**

### CODE REVIEW

File: docs/source/references/config_files/project/base_yml.rst
Comment (algomaster99) [inline docs/source/references/config_files/project/base_yml.rst:217]: - Capitalize "sentry" and link it to https://sentry.io/welcome/.
- Remove cms from here.

File: docs/source/references/config_files/project/base_yml.rst
Comment (algomaster99) [inline docs/source/references/config_files/project/base_yml.rst:331]: Omit this as this anyway an optional configuration.

Or I think it does throw an error when `integrations` is left blank. Find out why and open an issue about it. You may also submit a PR then.

File: docs/source/references/config_files/project/base_yml.rst
Comment (algomaster99) [inline docs/source/references/config_files/project/base_yml.rst:338]: 'smtp.example.com'

File: docs/source/references/config_files/project/base_yml.rst
Comment (algomaster99) [inline docs/source/references/config_files/project/base_yml.rst:341]: 'img@iitr.ac.in'

File: docs/source/references/config_files/project/base_yml.rst
Comment (algomaster99) [inline docs/source/references/config_files/project/base_yml.rst:342]: 'img@password'

File: docs/source/references/config_files/project/base_yml.rst
Comment (algomaster99) [inline docs/source/references/config_files/project/base_yml.rst:332]: Remove all this

File: docs/source/references/config_files/project/base_yml.rst
Comment (algomaster99) [inline docs/source/references/config_files/project/base_yml.rst:247]: The email ~account~ address from which all emails will be sent

File: docs/source/references/config_files/project/base_yml.rst
Comment (algomaster99) [inline docs/source/references/config_files/project/base_yml.rst:166]: I think this is for managing subscriptions in emails and notification. @pradumangoyal might be able to explain better.

File: docs/source/references/config_files/project/base_yml.rst
Comment (algomaster99) [inline docs/source/references/config_files/project/base_yml.rst:179]: I don't know what this is for but I feel it can be explained more clearly. @pradumangoyal can explain.

File: docs/source/references/config_files/project/base_yml.rst
Comment (pradumangoyal) [reply docs/source/references/config_files/project/base_yml.rst]: Don't omit this; this is not an optional field but can be considered as a required idea. An idea that how can one integrate 3rd party apps configuration through configuration files. If you think of any word can be used instead of integrations otherwise as it is finally parsed and used in the app on convenience. We have already configured sentry in the omniport accordingly to find if `integrations` key is there and have a sentry object. I would suggest to add this key with by default an empty dictionary as written in `base_stemcil.yml` for the sake of the propagation of this idea.

File: docs/source/references/config_files/project/base_yml.rst
Comment (pradumangoyal) [reply docs/source/references/config_files/project/base_yml.rst:179]: They are used to store and verify the verification tokens.

**Comment by pradumangoyal** [comment]
Ports are more often considered as strings; they are generally returned as strings by various methods in different languages. The reason may be https van also be passed as port in the sense localhost:https. So I think it would be better to revert those changes.

File: docs/source/references/config_files/project/base_yml.rst
Comment (pradumangoyal) [inline docs/source/references/config_files/project/base_yml.rst:337]: Email user is not maintainers but a separate email id of the site. Generally, these emails are no-reply emails which are not used for daily life communication but the purpose of the notification only. no-reply@omniport.com will propagate a better idea in my opinion.

File: docs/source/references/config_files/project/base_yml.rst
Comment (pradumangoyal) [reply docs/source/references/config_files/project/base_yml.rst:166]: Yes they are used to handle the notification subscriptions and objects

File: docs/source/references/config_files/project/base_yml.rst
Comment (pradumangoyal) [inline docs/source/references/config_files/project/base_yml.rst:219]: I think it will be better to mention the key :integrations: and its type as object

**Comment by pradumangoyal** [comment]
@nisarg73 Good that you updated the documentation it was much needed. Might be more reviews than a code PR. The reason being the docs are one and the only way of propagating our ideology to the other contributors currently. So there is a need to be extra careful.

**Comment by nisarg73** [comment]
> Ports are more often considered as strings; they are generally returned as strings by various methods in different languages. The reason may be https van also be passed as port in the sense localhost:https. So I think it would be better to revert those changes.

@dhruvkb ? :confused:

**Comment by algomaster99** [comment]
@nisarg73 is this done?

**Comment by algomaster99** [comment]
@nisarg73 any updates on this?

**Comment by nisarg73** [comment]
@algomaster99 I did ask @pradumangoyal to re-review this PR. Waiting for confirmation from his side.

**Review by pradumangoyal — APPROVED** [review]

**Pull request merged**
